receiver/kubeletstatsreceiver/README.md

```diff
@@ -285,3 +285,28 @@ rules:
     resources: ["nodes/proxy"]
     verbs: ["get"]
 ```
+
+### Warning about metrics' deprecation
+
+The following metrics will be removed in a future version:
+- `k8s.node.cpu.utilization` (replaced by `k8s.node.cpu.usage`)
+- `k8s.pod.cpu.utilization` (replaced by `k8s.pod.cpu.usage`)
+- `container.cpu.utilization` (replaced by `container.cpu.usage`)
+
+Users can use the following metrics instead:
+- `k8s.node.cpu.usage`
+- `k8s.pod.cpu.usage`
+- `container.cpu.usage`
```

Comment:
Since you already listed the `replaced by` in the previous section, I think this should be removed.